ccm_web/server/src/canvas/canvas.service.ts

```diff
@@ -23,11 +23,11 @@ type SupportedAPIEndpoint = '/api/v1/' | '/api/graphql/'
 
 const requestorOptions: GotOptions = {
   retry: {
-    limit: 4,
+    limit: 3,
     methods: ['POST', 'GET', 'PUT', 'DELETE'],
     statusCodes: got.defaults.options.retry.statusCodes.concat([403]),
     calculateDelay: ({ attemptCount, retryOptions, error, computedValue }) => {
-      const delay = computedValue === 0 ? 0 : 5000
+      const delay = computedValue === 0 ? 0 : 2000
```

Comment:
I just did this. My thinking is that we initially set these so high as a way to back off from the rate limit, and get all the request through, but that shouldn't be necessary anymore. We still want retrying behavior for intermittent Canvas errors and such, but lower levels like this should be sufficient.